pkg/cmdlets/prompts/License.go

```diff
@@ -0,0 +1,33 @@
+package prompts
+
+import (
+    "os"
+
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/prompt"
+)
+
+
+type offlineLicense struct {
+	LegalText
+    filepath string
+}
+
+func newOfflineLicense(filepath string) *offlineLicense {
+    return &offlineLicense{filepath: filepath}
+}
+
+func (license *offlineLicense) GetLegalText() (string, error) {
+    b, err := os.ReadFile(license.filepath)
+	if err != nil {
+		return "", errs.Wrap(err, "Unable to open TOS file")
+	}
+
+	return string(b), nil
+}
+
+func PromptOfflineLicense(out output.Outputer, prompt prompt.Prompter, filepath string)(bool,error) {
+    legalText := newOfflineLicense(filepath)
+    return PromptTOS(legalText,out,prompt)
```

Comment:
PromptTOS is prompting for a license? Again; these are different things. If we want to prompt for a license instead of a TOS then we should not funnel this through TOS specific logic.

I suspect we want a TOS though, and all this license stuff is a red herring.

Reply:
The installer is produced by the service and it's access to the service that requires a TOS.  The License is for the use of the runtime, regardless of how you received it and neither the installer nor runtime require access to the service.

Reply:
Note this particular comment is about intermixing TOS and License in the implementation. Regardless of which one is the right one, a license should not invoke something that's specific to a TOS, or vice versa.